.github/workflows/coding-standards.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - "*.x"
+      - entity-level-commit-order
```

Comment:
Isn't the plan to merge this PR and then remove that branch?

Reply:
Yes. I had those things in `.github/workflows` in mind, but thought we cannot safely remove them unless this PR is merged?

Reply:
Yes exactly… so now you should push a new commit to remove those, and the PR should be ready to merge, right?

Reply:
Did that. Surprised to see that tests etc. still being run for that new commit.

Reply:
D'oh. I get it. They run for the PR branch, no longer need to run for the other one.